lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -65,6 +65,40 @@ public function execute($arguments = [], $options = [])
             QubitTaxonomy::DESCRIPTION_DETAIL_LEVEL_ID => 'detailLevelTypes',
         ]);
 
+        // Define helper function used in post-save logic
+        $relateTermFunction = function ($import, $column, $taxonomyId) {
+            if (!empty($import->rowStatusVars[$column])) {
```

Comment:
Oops, I missed this big conditional last time @mcantelon.  I think it's better to exit the function rather than nest the body in a long conditional, i.e.:

```
if (empty($import->rowStatusVars[$column])) {
    return;
}
```

Reply:
Good thinking!